src/bin/cql-stress-cassandra-stress/operation/write.rs

```diff
@@ -43,6 +43,10 @@ impl CassandraStressOperation for WriteOperation {
     fn generate_row(&self, row_generator: &mut RowGenerator) -> Vec<CqlValue> {
         row_generator.generate_row()
     }
+
+    fn operation_tag(&self) -> &'static str {
+        "write"
```

Comment:
Same here as in the `src/bin/cql-stress-cassandra-stress/operation/mod.rs` module.
It doesn't match CS, moreover, CS has 2 variants, not 1 - for coordinated omission fixed case and for the affected.

Reply:
resolved